src/main/java/LongSet.java

```diff
@@ -3,7 +3,7 @@
 public class LongSet {
   public static void main (String[] args) {
     Set<Short> s = new HashSet<>();
-    for (short i = 0; i < 1000; i++) {
+    for (short i = 0; i < 10000; i++) {
```

Comment:
Good job! The static analysis tool Error Prone reported an error [1] used to be here, but you fixed it. Check out http://errorprone.info for more information.


[1] ```src/main/java/LongSet.java:8: error: [CollectionIncompatibleType] Argument 'i - 1' should not be passed to this method; its type int is not compatible with its collection's type argument Short
	  s.remove(i - 1);
	          ^
    (see http://errorprone.info/bugpattern/CollectionIncompatibleType)```